spec/features/user_visits_homepage_spec.rb

```diff
@@ -9,14 +9,14 @@
 
   scenario "successfully redirects to sign in page" do
     visit root_path
-    click_link 'Sign in'
+    click_button "Sign in"
```

Comment:
also, when you click the `Sign In` button it hits the `SessionsController#create` action.